### ISSUE TITLE
Just a freaking list - who cares for prices?

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#320]
I mean - I'm sure many people at home want to list their collection.
Or make a list for an idea, or share a list with someone else, for an exchange of list, or parts?

Then - even with the current default Buyer/collector layout:
* Prices (and even Cost) are shown,
* Items with no proce are displayed as Errors, while when people list 200 bricks, it's to list 200 bricks, there's no error there.

![image](https://user-images.githubusercontent.com/32078594/112392502-c97ba100-8cf9-11eb-81de-8fc1d30ea2eb.png)

Maybe we could have a 3rd default layout:
**BrickStore simple list (no price info)**

This would hide all price columns and would NOT show any absent price Error ?

![image](https://user-images.githubusercontent.com/32078594/112392557-e31ce880-8cf9-11eb-8696-a06b15802dce.png)

**Comment by henriquedesousa** [comment]
So you're rating about a free software that a developer offers his free time to you?

**Comment by rgriebl** [comment]
Please chill! @paramecie has probably spent more time than anyone else reporting bugs, thinking about new concepts plus doing the French translation.
THIS is another one of his ideas to make BrickStore more user friendly -- a good one I might add.

**Comment by BKuNL** [comment]
Good idea, I am also not very much interested in the price, just I want to know what I have. When I leave the price column out, I still have the errors.
Other possibility would be to fill in the price of 0,00, like it was possible in Brickstock.

**Comment by rgriebl** [comment]
I'm going through old issues: I'm aware that showing an error for a non-visible column is maybe a bad idea and should be fixed, but what's stopping you from just disabling "Show input errors"?

**Comment by BKuNL** [comment]
Well, to be honest, I have overlooked this possibility and have put it in place already.
Thanks for the tip.
 
Best regards/m.vr.gr.
Bert Kuyper

**Comment by paramecie** [comment]
> I'm going through old issues: I'm aware that showing an error for a non-visible column is maybe a bad idea and should be fixed, but what's stopping you from just disabling "Show input errors"?

1) It's tedious to hide all columns when a simple layout toggle would be easier as in the old BrickStore collector mode.
2) If you disable errors, then you won't see a qty of zero for example.

But sure, this is minor ;-)
